pkg/reconciler/subscription/subscription_test.go

```diff
@@ -117,6 +117,7 @@ func TestAllCases(t *testing.T) {
 			Name: "subscription, but subscriber is not addressable",
 			Objects: []runtime.Object{
 				NewSubscription(subscriptionName, testNS,
+					WithSubscriptionUID(subscriptionUID),
```

Comment:
why did the subscription tests need to change?

Reply:
Because I'm using the NewSubscription method from the broker test and I should not have the UID set in that case. The previous NewSubscription helper was coupled to the subscription test by setting the <name>-abc-123 as UID, which IMO should be configurable.